tests/integration/test_execute_databricks.py

```diff
@@ -0,0 +1,472 @@
+            Row(
+                l_orderkey_t=2,
+                l_partkey_t=22,
+                l_suppkey_t=222,
+                l_linenumber_t=2,
+                l_quantity_t=decimal.Decimal("2.0"),
+                l_extendedprice=decimal.Decimal("200.0"),
+                l_discount=decimal.Decimal("0.20"),
+                l_tax=decimal.Decimal("2.0"),
+                l_returnflag="A",
+                l_linestatus="F",
+                l_shipdate=datetime.strptime("2019-02-02", "%Y-%m-%d").date(),
+                l_commitdate=datetime.strptime("2019-02-05", "%Y-%m-%d").date(),
+                l_receiptdate=datetime.strptime("2019-02-04", "%Y-%m-%d").date(),
+                l_shipinstruct="DELIVER IN PERSON",
+                l_shipmode_t="MAIL",
+                l_comment_t="test",
+            ),
+            Row(
+                l_orderkey_t=3,
+                l_partkey_t=33,
+                l_suppkey_t=333,
+                l_linenumber_t=3,
+                l_quantity=decimal.Decimal("3.0"),
+                l_extendedprice=decimal.Decimal("300.0"),
+                l_discount=decimal.Decimal("0.35"),
+                l_tax=decimal.Decimal("3.0"),
+                l_returnflag="A",
+                l_linestatus="F",
+                l_shipdate=datetime.strptime("2019-03-01", "%Y-%m-%d").date(),
+                l_commitdate=datetime.strptime("2019-03-05", "%Y-%m-%d").date(),
+                l_receiptdate=datetime.strptime("2019-03-05", "%Y-%m-%d").date(),
+                l_shipinstruct="DELIVER IN PERSON",
+                l_shipmode_t="MAIL",
+                l_comment_t="test",
+            ),
+            Row(
+                l_orderkey_t=5,
+                l_partkey_t=55,
+                l_suppkey_t=555,
+                l_linenumber_t=5,
+                l_quantity=decimal.Decimal("5.0"),
+                l_extendedprice=decimal.Decimal("500.0"),
+                l_discount=decimal.Decimal("0.5"),
+                l_tax=decimal.Decimal("5.0"),
+                l_returnflag="A",
+                l_linestatus="F",
+                l_shipdate=datetime.strptime("2019-04-01", "%Y-%m-%d").date(),
+                l_commitdate=datetime.strptime("2019-05-05", "%Y-%m-%d").date(),
+                l_receiptdate=datetime.strptime("2019-05-04", "%Y-%m-%d").date(),
+                l_shipinstruct="DELIVER IN PERSON",
+                l_shipmode_t="MAIL",
+                l_comment_t="test",
+            ),
+        ],
+        schema=tgt_schema,
+    )
+
+    src_data.write.format("delta").mode("overwrite").saveAsTable(
+        f"{test_config.db_mock_catalog}." f"{test_config.db_mock_schema}.{test_config.db_mock_src}"
+    )
+    tgt_data.write.format("delta").mode("overwrite").saveAsTable(
+        f"{test_config.db_mock_catalog}." f"{test_config.db_mock_schema}.{test_config.db_mock_tgt}"
+    )
+
+
+def test_execute_report_type_is_data_with_all_match(setup_databricks_src, spark, ws, test_config, reconcile_config):
```

Comment:
where are all the scenarios we are testing?

Reply:
It has all four report types and has tested all the configurations. Let me know if anything needs to be added or if something should be covered.

Reply:
we should be testing other configuration like scenarios, complex types, thresholds, transformation, filters etc.